src/platforms/ctxlink/WiFi_Server.c

```diff
@@ -844,7 +975,7 @@ static void app_socket_callback(SOCKET sock, uint8_t msg_type, void *msg)
 				// Copy data to circular input buffer
 				//
 				for (int16_t i = 0; local_count != 0;
-					 i++, local_count--, input_index = (input_index + 1) % INPUT_BUFFER_SIZE) {
+					i++, local_count--, input_index = (input_index + 1) % INPUT_BUFFER_SIZE) {
```

Comment:
`clang-format` wants its space back here - likely this is older vs newer clang-format and a bug being triggered in the older. We'll get the version of `clang-format` upgraded in CI once v2.0 is released.

Reply:
While this is fixed, it is bound to reoccur if I work on this file again. When I save the file clang-format runs and re-fixes it for me. So, have the CI scripts deal with it correctly is kind of important.